core/merginapi.cpp

```diff
@@ -2359,7 +2477,7 @@ ProjectDiff MerginApi::compareProjectFiles( const QList<MerginFile> &oldServerFi
       if ( chkOld == chkLocal )
       {
         // R-D
-        diff.remoteDeleted << filePath;
+        diff.remoteDeleted << filePath; // here (ak si to nikdy nemal, ani to nemaz) -> skusim to bez toho
```

Comment:
let's translate the comment to English :-)

Reply:
my bad 🤦🏻‍♂️